docs/source/tagset/elemento-pub-date.rst

```diff
@@ -3,29 +3,27 @@
 <pub-date>
 ----------
 
-Aparece em
+Aparece em:
+
   :ref:`elemento-article-meta`
- 
-Atributos obrigatórios
-  1. pub-type='epub' ou pub-type='epub-ppub'
- 
-Ocorre
+
+Atributos obrigatórios:
+
+  1. ``@pub-type='epub'`` ou ``@pub-type='epub-ppub'``
+
+Ocorre:
+
   Uma vez
 
 
-Para a marcação da data de publicação do artigo/número utiliza-se a 
-tag ``<pub-date>`` a qual pode conter os elementos :ref:`elemento-day`, 
-:ref:`elemento-month`, :ref:`elemento-season` e obrigatoriamente 
-:ref:`elemento-year`. Esta tag deve estar acompanhada do atributo ``@pub-type``.
- 
-A data de publicação pode ser do tipo ``epub-ppub`` se houver uma versão 
-impressa do número, apenas ``epub`` para publicação digital ou em 
-``ahead-of-print``.
- 
+A data de publicação do artigo/número utiliza o elemento ``<pub-date>``, o qual pode conter os elementos :ref:`elemento-day`, :ref:`elemento-month`, :ref:`elemento-season` e, obrigatoriamente, :ref:`elemento-year` que deve estar acompanhada do atributo ``@pub-type``.
+
+``@pub-type`` pode ser ``epub-ppub`` se houver uma versão impressa do número ou apenas ``epub`` para publicação digital ou em ``ahead-of-print``.
```

Comment:
_ahead-of-print_ deveria ser um termo no glossário e não um texto em fonte monoespaçada.